addons/web/static/tests/core/utils/numbers.test.js

```diff
@@ -330,4 +331,24 @@ describe("formatFloat", () => {
         expect(formatFloat(1.0045e22, options)).toBe("1.005e+22");
         expect(formatFloat(-1.0045e22, options)).toBe("-1.004e+22");
     });
+
+    test("toFixed VS formatFixedDecimals", () => {
```

Comment:
I wouldn't export formatFixedDecimals, because people should not use it directly. So I wouldn't test it by itself either. Instead, I would simply add an assertion in formatFloat and humanReadable tests, with a value like 2.925, which before your fix rounded up to 2.92 and after 2.93. To better highlight the relevance of the assertion, I would also asset 1.925, to show that both round up away from zero.